src/Chainweb/Version.hs

```diff
@@ -925,21 +925,22 @@ chainweb213Pact Development = (>= 95)
 chainweb213Pact (FastTimedCPM g) | g == petersonChainGraph = (> 25)
 chainweb213Pact _ = const True
 
+
 -- | Pact and coin contract changes for Chainweb 2.14
 --
 chainweb214Pact
     :: AtOrAfter
     -> ChainwebVersion
     -> BlockHeight
     -> Bool
-chainweb214Pact aoa v h = case aoa of
+chainweb214Pact  aoa v h = case aoa of
     At -> go (==) v h
     After -> go (flip (>)) v h
   where
-    go f Mainnet01 = f 2605663 -- 2022-04-22T00:00:00Z
-    go f Testnet04 = f 2134331 -- 2022-04-21T12:00:00Z
-    go f Development = f 115
-    go f (FastTimedCPM g) | g == petersonChainGraph = f 30
+    go f Mainnet01 = f 2605696 -- 2022-04-22 00:00:13
+    go f Testnet04 = f 2112766 -- 2022-04-14 00:00:43
+    go f Development = f 100
+    go f (FastTimedCPM g) | g == petersonChainGraph = (f 27)
```

Comment:
What is the reason for these changes? Maybe just a git merge glitch?

Reply:
oh I did not mean to change this. Merge glitch.